docs/Contributing/dev-env.rst

```diff
@@ -91,6 +91,8 @@ your computer:
 .. code-block:: sh
 
   ~/GitHub/freeciv21$ git remote add upstream https://github.com/longturn/freeciv21.git
+  ~/GitHub/freeciv21$ git fetch upstream
```

Comment:
`git pull upstream master` includes the `fetch`, so this step could be dropped.

Reply:
I don't think so. You have to `fetch` to get any new tags and such. `pull` doesn't do that on its own.

Reply:
Ah right, the `pull` probably fetches only the `master` branch and related tags in this example. I stand corrected :)